pypeman/contrib/time.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+import datetime
+
+from pypeman import channels, message
+
+from aiocron import crontab
+
+class TimeChannel(channels.BaseChannel):
```

Comment:
better name might be TimedChannel 
TimeChannel sounds aas if the contesnts of the channel were a time